src/helpers/Api/Api.js

```diff
@@ -55,6 +56,8 @@ class Api {
         let promise = new Promise((resolve, reject) => {
           request = superagent[method](formatUrl(path));
 
+          debug('requests')(formatUrl(path));
```

Comment:
I usually use `DEBUG=superagent yarn start`. Would that be enough?

Reply:
you are right... I didn't remember this one! This one was a quick, compulsive PR :)